packages-next/fields/src/types/relationship/index.ts

```diff
@@ -44,6 +44,7 @@ export type RelationshipFieldConfig<
     hideCreate?: boolean;
   };
   defaultValue?: FieldDefaultValue<Record<string, unknown>>;
+  // FIXME: NOT IMPLEMENTED
```

Comment:
This feature isn't actually implemented, so I've not included it in the API docs. I'm going to follow up on what to do with the types themselves in the context of the upcoming field changes separate to this PR.